src/ui/Interface.js

```diff
@@ -19,8 +19,14 @@ module.exports = function Interface (options) {
 
   panMapWhenInputsChange();
 
+  function obscureLocation() {
```

Comment:
I think that makes this line unnecessary now -- looking good though! Almost there.